paasta_tools/instance/kubernetes.py

```diff
@@ -46,8 +46,6 @@
 
 
 INSTANCE_TYPES_CR = {"flink", "cassandracluster", "kafkacluster"}
-# TODO: remove "cassandracluster" from INSTANCE_TYPES_K8S after
-# print_cassandra_status proves to be stable.
```

Comment:
I had added this comments previously, before we decided to print k8s status metadata as well. It i irrelevant now.